src/lib/components/FileUploader/FileUploaderArea.js

```diff
@@ -148,7 +147,7 @@ const FileTableRow = ({
                 link
                 className="action"
                 name="trash alternate outline"
-                color="blue"
+                className="primary"
```

Comment:
Note that `className` is duplicated.
```suggestion
                className="action primary"
```

Reply:
ah woops, will update! Thanks a lot!